classifier/requirements.txt

```diff
@@ -6,5 +6,4 @@ numpy==1.15.4
 pandas==0.23.4 
 scikit-learn==0.20.1 
 tensorflow==1.12.0
-
-  
+pyyml==0.0.2
```

Comment:
AFAICT the standard package for yaml in python is PyYaml:

https://pypi.org/project/PyYAML/

so in requirements.txt would be

```
PyYAML==5.1.2
```